tools/pfe-tools/dev-server/config.ts

```diff
@@ -116,6 +116,19 @@ function pfeDevServerPlugin(options: PfeDevServerInternalConfig): Plugin {
             const { element, fileName } = ctx.params;
             ctx.redirect(`/${elementsDir}/${element}/${fileName}.ts`);
           })
+          // Redirect `components/pf-jazz-hands|jazz-hands/demo/*-lightdom.css` to `components/pf-jazz-hands/*-lightdom.css`
+          .get(`/${componentSubpath}/:element/demo/:demoSubDir?/:fileName.css`, async (ctx, next) => {
```

Comment:
instead of adding a new case, can we modify the existing case at main branch's line 120? (the block below this one)